flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/ElasticsearchSinkFunction.java

```diff
@@ -65,6 +65,11 @@
 	 */
 	default void open() {}
 
+	/**
+	 * Tear-down method for the function. It is called when the sink closes.
+	 */
+	default void close () throws Exception {}
```

Comment:
It's a bit weird that the `open` function does not allow to throw an exception. I admit it is out of the scope of this PR though.

Reply:
I can update `open` to `default void open() throws Exception {}` real quick. It's a backward compatible change and should have no user impact.

Reply:
Sounds good to me.